lib/slurm_squeue_client.rb

```diff
@@ -17,6 +17,7 @@ def initialize(cluster)
     end
   
     @cluster_id = cluster.id
+    @canonical_cluster_id = @cluster_id.to_s.partition('-').first
```

Comment:
The cluster configs have this for Slurm.

```
  job:
    adapter: "slurm"
    cluster: "pitzer"
    host: "pitzer-slurm01.ten.osc.edu"
```

```
  job:
    adapter: "slurm"
    cluster: "owens"
    host: "owens-slurm01.ten.osc.edu"
```

So you use `cluster.job_config[:cluster]`. And if that is nil i.e not set, you don't append `--clusters=` to the commands.